sdk/synapse/Azure.Analytics.Synapse.AccessControl/README.md

```diff
@@ -98,8 +108,8 @@ RoleDefinitionsClient definitionsClient = new RoleDefinitionsClient(new Uri(endp
 First, you need to the determine the ID of the role you wish to assign, along with the ID of the principal you wish to assign that role.
 
 ```C# Snippet:PrepCreateRoleAssignment
-Response roleDefinitionsReponse = definitionsClient.GetRoleDefinitions(new());
-BinaryData roleDefinitionsContent = roleDefinitionsReponse.Content;
+Response roleDefinitionsResponse = definitionsClient.GetRoleDefinitions(new());
```

Comment:
Given this is extracted, the source would have to be updated as well (or was it wrong and just never flagged)?